geode-core/src/main/java/org/apache/geode/internal/cache/persistence/PersistenceAdvisorImpl.java

```diff
@@ -533,7 +535,8 @@ public boolean checkMyStateOnMembers(Set<InternalDistributedMember> replicates)
         String message = String.format(
             "Region %s remote member %s with persistent data %s was not part of the same distributed system as the local data from %s",
             regionPath, member, remoteId, myId);
-        throw new ConflictingPersistentDataException(message);
+        logger.warn(message);
```

Comment:
How about `logger.error(message)`?

Reply:
it's not an error. It's race condition cause of the member to be temporarily out of sync. Geode can auto fix it. We just don't need this out-of-sync member to be a GII provider candidate.

Reply:
A warning message will cause customers and support to think something is wrong. Is the user suppose to take same action based on this message? If you are correct that this is something geode will deal with automatically then at the most this should be an "info" message or "debug".

Reply:
OK, I will change to info